core/src/main/scala/cats/data/OptionT.scala

```diff
@@ -62,6 +62,16 @@ final case class OptionT[F[_], A](value: F[Option[A]]) {
    * Catamorphism on the Option. This is identical to [[fold]], but it only has
    * one parameter list, which can result in better type inference in some
    * contexts.
+   *
+   * Example:
+   * {{{
+   * scala> import cats.implicits._
```

Comment:
`cats.implicits` is deprecated.

```suggestion
   * scala> import cats.syntax.all._
```

Reply:
Thanks, that was news to me. There were actually a bunch of unused `cats.implicits._` imports in the Doctests.
I removed them.